### DISCUSSION TITLE
Renovate fails to update the package-lock.json resulting in ERESOLVE error

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [renovatebot/renovate#8873] [Request Help] [answered]
**What Renovate type, platform and version are you using?**

GitHub: https://github.com/azerothcore/Keira3

**Describe the bug**

Renovate seems no longer updating the `package-lock.json` file, resulting in an error. See:

https://github.com/azerothcore/Keira3/pull/1092

**Relevant debug logs**

Check the above link to see the full error log.

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] This is a really small bug, it does not need a reproduction (think small typo)
- [ ] I have provided a minimal reproduction repository
- [x] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Comment by rarkins** [comment, marked as answer]
This is an npm v7 problem that you need to resolve manually. Npm's error message is quite descriptive.

**Reply by FrancescoBorzi** [reply]
@rarkins I did it but it looks like the bot is still not updating the package-lock files for the new versions.

**Reply by viceice** [reply]
@FrancescoBorzi Please share the debug logs from the run when the commit of the pr was made.

You will find the link in the pr footer
![image](https://user-images.githubusercontent.com/1798109/109280276-15cae280-781b-11eb-8c92-593817b996f5.png)

**Reply by viceice** [reply]
Additionally try a rerun with this checkbox:
![image](https://user-images.githubusercontent.com/1798109/109280379-3eeb7300-781b-11eb-888b-9b6f7b9272a1.png)

**Reply by FrancescoBorzi** [reply]
> @FrancescoBorzi Please share the debug logs from the run when the commit of the pr was made.
> 
> You will find the link in the pr footer
> ![image](https://user-images.githubusercontent.com/1798109/109280276-15cae280-781b-11eb-8c92-593817b996f5.png)

https://app.renovatebot.com/dashboard#github/azerothcore/Keira3

**Reply by viceice** [reply]
@FrancescoBorzi We can't see that, only owner and maintainer or your repo can see your logs

**Reply by FrancescoBorzi** [reply]
https://paste.ubuntu.com/p/4C5Wz7KXpy/

**Reply by rarkins** [reply]
Are you familiar with the differences between how npm 6 and npm 7 treat peerDependencies?

If yes, then please explain what specifically you think Renovate should be doing differently.

If no, I suggest you downgrade to npm 6 or enable the legacy peer deps handling in your `.npmrc`.
